src/test/regress/sql/propagate_extension_commands.sql

```diff
@@ -1,7 +1,3 @@
--- print whether we're using version > 12 to make version-specific tests clear
```

Comment:
do we still need expected/propagate_extension_commands_1.out (or expected/propagate_extension_commands.out). Would be a good cleanup if we can remove the output files al together for these cases